_observing-your-data/query-insights/query-insights-dashboard.md

```diff
@@ -78,22 +78,40 @@ The **Refresh** button reloads the query data based on the selected filters and
 
 ### F. Metrics table
 
-The metrics table displays the following metrics for each query.
-
-| Metric                  | Description                                                                 |
-|-------------------------|-----------------------------------------------------------------------------|
-| **ID**                  | The unique identifier for the query.                                         |
-| **Type**                | The type of query, such as `query`or `group`.                               |
-| **Query Count**         | The number of times that the query has been executed.                            |
-| **Timestamp**           | When the query was run.                                                     |
-| **Latency**             | The time taken for the query to execute.                                    |
-| **CPU Time**            | The CPU resources consumed by the query.                                    |
-| **Memory Usage**        | The memory usage of the query.                                              |
-| **Indexes**             | The index or indexes on which the query was executed.                       |
-| **Search Type**         | The type of search used, for example, `query then fetch`.                   |
-| **Coordinator Node ID** | The node that coordinated the query.                                        |
-| **Total Shards**        | The total number of shards involved in running the query.                    |
+The metrics table dynamically adapts based on your **Type** filter selection (**Query**, **Group**, or both). Dynamic columns improve clarity by showing only the relevant data for each query type.
 
+When you select **queries only**, the table displays individual metrics, including **Latency**, **CPU Time**, and **Memory Usage**. The **Query Count** column isn't displayed because each row represents a single query, as shown in the following image:
```

Comment:
```suggestion
When you select **queries only**, the table displays individual metrics, including **Latency**, **CPU Time**, and **Memory Usage**. The **Query Count** column isn't displayed because each row represents a single query, as shown in the following image.
```